### PR TITLE
adds an iff-less heavy machinegun

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
adds a heavy machinegun that has no iff, not to marine vendors yet though i'd like to add it as a even heavier counterpart that has no iff for chokepoint holding to the HSG, considering it has no IFF, and aimmode is broken on mounted guns.

## Why It's Good For The Game
Adds more interesting stuff to the marine arsenal.

## Changelog
:cl:
expansion: Adds an IFF-less heavy machinegun to the game, not in yet for acquireable players.
/:cl:

### CODE REVIEW

**Comment by SilencedMP5A5** [comment]
show sprite

**Comment by AnCopper** [comment]
> show sprite

image

**Comment by lbnesquik** [comment]
Is this a World War one machine gun?

**Review by TiviPlus — CHANGES_REQUESTED** [review]
The sprite is just an obvious copypaste of the hmg make it better

**Comment by Pariah919** [comment]
@TiviPlus i changed the tripod
image

**Comment by BraveMole** [comment]
"not in yet for acquireable players" Why? :(

**Comment by Tristan63** [comment]
mg08 aircraft machinegun?

File: code/modules/projectiles/guns/mounted.dm [hunk 1]
```diff
@@ -108,3 +108,39 @@
 
 	flags_gun_features = GUN_AUTO_EJECTOR|GUN_AMMO_COUNTER|GUN_LOAD_INTO_CHAMBER
 
+// This is a deployed IFF-less mACHINEGUN, has 500 rounds, drums do not fit anywhere but your belt slot and your back slot. But it has 500 rounds. That's nice.
+
+/obj/item/weapon/gun/mg08
+	name = "\improper MG-08/495 heavy machinegun"
```
Comment (TiviPlus) [inline code/modules/projectiles/guns/mounted.dm:114]: dont use numbers in path names please

File: code/modules/projectiles/magazines/mounted.dm [hunk 2]
```diff
@@ -16,3 +16,17 @@
 /obj/item/ammo_magazine/tl102/hsg_nest
 	max_rounds = 1500
 	gun_type = /obj/item/weapon/gun/tl102/hsg_nest
+
+/obj/item/ammo_magazine/mg08
```
Comment (TiviPlus) [inline code/modules/projectiles/magazines/mounted.dm:20]: Ditto

**Comment by Losenis** [comment]
Finally. Now we can put down marines blocking benos in a few shots instead of having to dump a burst of 20 into them with a GPMG.

**Comment by lbnesquik** [comment]
No way to spawn this gun

**Review by TiviPlus — APPROVED** [review]

File: code/modules/projectiles/gun_attachables.dm [hunk 3]
```diff
@@ -741,6 +741,15 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	desc = "A rail mounted zoom sight scope specialized for the T-127 sniper rifle. Allows zoom by activating the attachment. Use F12 if your HUD doesn't come back."
 	flags_attach_features = ATTACH_ACTIVATION
 
+/obj/item/attachable/scope/unremovable/mg08
+	name = "MG-08/495 long range ironsights"
```
Comment (TiviPlus) [inline code/modules/projectiles/gun_attachables.dm:745]: you forgot thsi typepath

**Pull request merged**